gems/aws-sdk-resources/VERSION

```diff
@@ -1 +1 @@
-3.0.0.rc1
+3.0.0.rc8
```

Comment:
This isn't really right - we should reverse the code, but the release count is still the release count.